bundles/tampere/search-from-channels/instance.js

```diff
@@ -806,16 +806,27 @@ Oskari.clazz.define(
          */
         _getVectorLayerStyle: function(){
 
-            var style = OpenLayers.Util.applyDefaults(style, OpenLayers.Feature.Vector.style['default']);
-            style.pointRadius = 8;
-            style.strokeColor = '#D3BB1B';
-            style.fillColor = '#FFDE00';
-            style.fillOpacity = 0.6;
-            style.strokeOpacity = 0.8;
-            style.strokeWidth = 2;
-            style.cursor = 'pointer';
-
-            return style;
+            var featureStyle = {
+                fill: {
+                  color: 'rgb(153,204,0,0.3)',
+                },
+                stroke: {
+                  color: '#FF0000',
+                  width: 10
+                },
+                text : {
+                  scale : 1.3,
+                  fill : {
+                    color : 'rgba(0,0,0,1)'
+                  },
+                  stroke : {
+                    color : 'rgba(255,255,255,0.8)',
+                    width : 2
+                  },
+                  labelProperty: 'test_property'
```

Comment:
Seems you don’t have an attribute named ’test_property’ for the features so this styling can be removed.